sec_certs/settings-schema.json

```diff
@@ -0,0 +1,123 @@
+{
+    "title": "settings for sec-certs",
+    "type": "object",
+    "properties": {
+        "log_filepath": {
+            "type": "object",
```

Comment:
I suppose it has to be an `object` because it is of the form

```
  description: something
  value: some value
```

In that case, you can define your own type and say that it must contain only these two fields, that the first field should be string, second field arbitrary. E.g., see https://stackoverflow.com/questions/32263648/how-to-define-custom-object-type-in-json-schema and https://json-schema.org/understanding-json-schema/structuring.html#defs

Therefore, all that is typed `object` now, can be turned into custom type that is more fitting for our case.